NHE-1171: Declare SRIOVFailedToConfigureVF risk

Davoska opened the pull request:
The corresponding impact statement: https://issues.redhat.com/browse/NHE-1171

Versions affected at the moment:
- 4.14.34+
- 4.15.25+

Created `4.14.34-SRIOVFailedToConfigureVF.yaml` manually.

Then run the following command for the next versions:
```
curl -s 'https://api.openshift.com/api/upgrades_info/graph?channel=candidate-4.14&arch=amd64' | jq -r '.nodes[] | .version' | grep '^4[.]14[.]\(3[5-9]\|4[0-9]\)$' | while read VERSION; do sed "s/4.14.34/${VERSION}/" blocked-edges/4.14.34-SRIOVFailedToConfigureVF.yaml > "blocked-edges/${VERSION}-SRIOVFailedToConfigureVF.yaml"; done
```

Created `4.15.25-SRIOVFailedToConfigureVF.yaml` manually.

Then run the following command for the next versions:
```
curl -s 'https://api.openshift.com/api/upgrades_info/graph?channel=candidate-4.15&arch=amd64' | jq -r '.nodes[] | .version' | grep '^4[.]15[.]\(2[6-9]\|3[0-9]\)$' | while read VERSION; do sed "s/4.15.25/${VERSION}/" blocked-edges/4.15.25-SRIOVFailedToConfigureVF.yaml > "blocked-edges/${VERSION}-SRIOVFailedToConfigureVF.yaml"; done
```

Davoska commented on blocked-edges/4.14.33-SRIOVFailedToConfigureVF.yaml:
```diff
@@ -0,0 +1,15 @@
+to: 4.14.33
+from: ^4[.](13[.].*|14[.]([0-9]|1[0-9]|2[0-9]|3[0-2]))[+].*$
+url: https://issues.redhat.com/browse/NHE-1171
+name: SRIOVFailedToConfigureVF
+message: |-
+  Pods using a secondary interface of SRI-OV VF will stop coming up.
```
I am not sure of the exact best wording at the moment as an impact statement is not proposed, and I do not understand the issue too much.

Davoska replied:
The message was updated to `On clusters with the SR-IOV Network Operator installed and configured, pods with a secondary interface of SRI-OV VF will fail to create a pod sandbox and thus will not function.`.

Davoska commented:
Steps for testing:

1. Launched a cluster using `launch 4.16.5 aws`
2. The cluster is fresh and does not contain the SR-IOV Network Operator.
3. Run the PromQL query:
![image](https://github.com/user-attachments/assets/3f76bd85-9783-4108-90c2-6dc1acd2dd3d)
5. Install the SR-IOV Network Operator from the OperatorHub.
6. Run the PromQL query:
![image](https://github.com/user-attachments/assets/04348f17-d995-4b30-bd63-811a93752337)

wking commented on blocked-edges/4.14.33-SRIOVFailedToConfigureVF.yaml:
```diff
@@ -0,0 +1,15 @@
+- type: PromQL
+  promql:
+    promql: |
+      (
```
nit: no need for the wrapping parens, because there's nothing else that we're grouping against.

wking replied:
Fixed in c5b088415a6e9bd0 -> 1dddc4dec09c2acd94; I'm marking this thread resolved.

Davoska replied:
Done.

Davoska commented:
In the latest commit https://github.com/openshift/cincinnati-graph-data/commit/bdd81c17ae4f9ca4e787674c0cb99114e0f7bd0c, I have updated the affected versions and updated the message.

sdodson commented:
/lgtm

sdodson commented:
/override ci/prow/e2e
